package.json

```diff
@@ -30,16 +30,19 @@
     "@dcl/single-sign-on-client": "^0.1.0",
     "@dcl/ui-env": "^1.4.0",
     "@magic-ext/oauth": "^15.1.1",
+    "@well-known-components/fetch-component": "^2.0.2",
     "ajv": "^8.12.0",
     "classnames": "^2.3.2",
+    "dcl-catalyst-client": "^21.5.5",
     "decentraland-connect": "^5.2.4",
     "decentraland-ui": "^5.2.0",
     "ethers": "^6.8.0",
     "magic-sdk": "^21.2.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-router-dom": "^6.17.0",
-    "socket.io-client": "^4.7.2"
+    "socket.io-client": "^4.7.2",
+    "vite-plugin-node-polyfills": "^0.19.0"
```

Comment:
Required to use node's crypto lib. Required by the catalyst client when building the profile entity.